helper.linux.fish

```diff
@@ -737,5 +745,64 @@ function makeSnippets
   and buildTarGzSnippet
 end
 
+function copyReleaseEdition
+  set -l dest $argv[1]
+  set -l repo $argv[2]
+  set -l edition $argv[3]
+  set -l l $argv[4]
+
+  # debian
```

Comment:
shouldn't there be a central place where we calculate the filenames?

Reply:
Good point, will add